src/components/FriendsItemsBox.vue

```diff
@@ -113,16 +117,16 @@
                 });
             },
             addItem() {
-                this.selectedFriends.forEach(function (item) {
+                this.selectedFriends.forEach(friend => {
```

Comment:
name this a little more suggestive `selectedFriends` -> `selectedFriendIds` and `friend` -> `friendId`